utilities/ovs-p4ctl.in

```diff
@@ -953,24 +953,35 @@ def validate_args(argv, command, expected_nr):
         raise Exception("ovs-p4ctl: '{}' command requires at least {} "
                         "arguments".format(command, expected_nr))
 
-
+grpc_cl = ""
 def main():
+    global grpc_cl
     if len(sys.argv) < 2:
         print("ovs-p4ctl: missing command name; use --help for help")
         sys.exit(1)
-    parser = argparse.ArgumentParser(usage=USAGE)
-    parser.add_argument('command', help='Subcommand to run')
 
-    args = parser.parse_args(sys.argv[1:2])
-    if args.command not in all_commands.keys():
+    parser = argparse.ArgumentParser(usage=USAGE)
+    parser.add_argument('-g', '--grpcaddr', required=False, type=str, help="Grpc server ip:port xxx.xxx.xxx.xxx:yyyy")
+    parser.add_argument('command', nargs='*', help='Subcommand to run')
```

Comment:
So domain names are not supported? Will this be added later?

Reply:
As of now, Domain names aren't supported.